### DISCUSSION TITLE
Network plugin listener send multiple logs on Android

### DISCUSSION THREAD

**Discussion by petrot** [ionic-team/capacitor#6702] [General]
We get multiple logs on our console from the Network plugin. I don't know if this is a Capacitor plugin bug or just a normal Android behaviour.. It happens on Android device, connected to a PC. No user interaction happened, the logs are just coming, but there aren't real changes in the status... What do you think? Is it a bug?

![IMG_5816](https://github.com/ionic-team/capacitor/assets/4799473/c5f861b9-399d-4fa7-b35b-bd306ab2dba6)

**Comment by kasik96** [comment]
Same problem.

![image](https://github.com/ionic-team/capacitor/assets/7235992/0fb3bbf3-02e5-4ea2-8711-5b495e8bc45d)

**Comment by Theqwertypusher** [comment]
Any luck on this issue? I have the same problem 
![image](https://github.com/user-attachments/assets/13ed313b-5bbd-4598-b287-d0707cc7f878)
